src/argus_htmx/apps.py

```diff
@@ -1,7 +1,11 @@
+import pathlib
 from django.apps import AppConfig
 
 
 class HtmxFrontendConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     label = "argus_htmx"
     name = "argus_htmx"
+
+    def tailwind_css_files(self):
+        yield from pathlib.Path(__file__).parent.glob("tailwindtheme/snippets/*.css")
```

Comment:
It could be, technically, but since it's the reference/example implementation, keeping it as an instance method conveys to third party developers that it can be either an instance method, classmethod or staticmethod. Making it a staticmethod loses that. For me it was non-obvious whether django instantiates the appconfig (and thus we can use normal methods) or uses the class directly, until I tried it out.